.github/workflows/test.yml

```
@@ -75,12 +75,10 @@ jobs:
             # Upload coverage reports to Codecov
             - name: Upload coverage to Codecov
               uses: codecov/codecov-action@v1.0.10
-              with: 
-                yml: ./codecov.yml
 
     rpcTest:
         name: RPC Generator Tests
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-18.04
```

```suggestion
        runs-on: ubuntu-20.04
```

Will this work on the latest LTS?

---

It does, without displaying any warning